static/app/components/tooltip.tsx

```diff
@@ -1,5 +1,6 @@
 import {
   cloneElement,
+  CSSProperties,
```

Comment:
Wait is this actually a regression? Why can't we use the React.* types?

Reply:
No, this one is not, but I changed it when testing to see if it had an impact, just didnt revert